src/compiler/sys.ts

```diff
@@ -1620,12 +1619,6 @@ namespace ts {
                             options = { persistent: true };
                         }
                     }
-

```

Comment:
I understand a full revert is safer at this point in the cycle, but it would be nice to keep (or quickly restore) this part of the change.

Reply:
Can you review the latest commit and see if this is what you had in mind?